src/bindings/PyDP/pydp_lib/algorithm_builder.hpp

```diff
@@ -222,19 +223,24 @@ class AlgorithmBuilder {
         throw std::runtime_error(result.status().ToString());
       }
       if constexpr ((should_return_T<T, Algorithm>()))
-        return dp::GetValue<T>(result.ValueOrDie());
+        return dp::GetValue<T>(result.value());
       if constexpr ((should_return_double<T, Algorithm>()))
-        return dp::GetValue<double>(result.ValueOrDie());
+        return dp::GetValue<double>(result.value());
       if constexpr ((should_return_int<T, Algorithm>()))
-        return dp::GetValue<int64_t>(result.ValueOrDie());
+        return dp::GetValue<int64_t>(result.value());
     });
 
     // Other methods
     pyself.def("reset", &Algorithm::Reset);
 
     pyself.def("serialize", &Algorithm::Serialize);
 
-    pyself.def("merge", &Algorithm::Merge);
+    pyself.def("merge", [](Algorithm& pythis, const dp::Summary& summary) {
```

Comment:
returning status is not Python style, it's better to raise an exception (as it's already done in other places)

Reply:
is `status.toString()` user friendly?

Reply:
Yes (at least it's as friendly as for C++ library)
Example:  **INTERNAL: Bounded mean summary must have exactly one pos_sum but got 32**